.github/workflows/ci-build.yaml

```
@@ -0,0 +1,29 @@
+name: CI build
+on:
+  pull_request: {}
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: golang:1.16.10-alpine3.14
+    
+    steps:
+      - name: Setup environment
```

```suggestion
      - name: Set up environment
```
nit: "set up" is the verb, "setup" is a noun